operate/workload-isolation-interaction.mdx

```diff
@@ -45,6 +45,16 @@ Resource groups offer a powerful mechanism for fine-grained control over compute
 
 A *resource group* is a named logical pool consisting of specific compute nodes (CNs) within your RisingWave cluster. Administrators typically define these groups and their associated CNs during cluster setup or configuration.
 
+### Defining a resource group
+
+Resource groups are defined by assigning compute nodes to them at startup. Each compute node can be independently assigned to a named group, and all workers launched with the same resource group name collectively form that logical group.
+
+This assignment is done when launching a worker by using the `--resource-group` command-line flag or by setting the `RW_RESOURCE_GROUP` environment variable.
```

Comment:
@shanicky Can we provide an example here? I don't know how to launch/start a compute node. And by using the enrironmental variable approach, when this compute node is started, it'll be automatically running on the assigned resource group, right? 

And here "worker" = "compute node", I assume.